From ben@decadent.org.uk Fri Jun 25 02:35:49 2010
Return-Path: <devel-bounces@linuxdriverproject.org>
X-Flags: 0000
Delivered-To: GMX delivery to s.l-h@gmx.de
Received: (qmail invoked by alias); 25 Jun 2010 00:37:02 -0000
Received: from driverdev.linuxdriverproject.org (EHLO driverdev.linuxdriverproject.org) [140.211.166.17]
  by mx0.gmx.net (mx095) with SMTP; 25 Jun 2010 02:37:02 +0200
Received: from driverdev.linuxdriverproject.org (localhost [127.0.0.1])
	by driverdev.linuxdriverproject.org (Postfix) with ESMTP id 0724E9893C;
	Thu, 24 Jun 2010 17:41:35 -0700 (PDT)
X-Original-To: devel@driverdev.osuosl.org
Delivered-To: devel@driverdev.osuosl.org
Received: from shadbolt.e.decadent.org.uk (shadbolt.e.decadent.org.uk
	[88.96.1.126])
	by driverdev.linuxdriverproject.org (Postfix) with ESMTP id 5185E46CCB
	for <devel@driverdev.osuosl.org>; Thu, 24 Jun 2010 17:40:27 -0700 (PDT)
Received: from [192.168.4.185] (helo=localhost)
	by shadbolt.decadent.org.uk with esmtps
	(TLS1.0:RSA_AES_256_CBC_SHA1:32) (Exim 4.69)
	(envelope-from <ben@decadent.org.uk>)
	id 1ORwta-0002hT-RS; Fri, 25 Jun 2010 01:35:51 +0100
Received: from ben by localhost with local (Exim 4.72)
	(envelope-from <ben@decadent.org.uk>)
	id 1ORwtZ-000228-TA; Fri, 25 Jun 2010 01:35:49 +0100
From: Ben Hutchings <ben@decadent.org.uk>
To: Greg KH <greg@kroah.com>
In-Reply-To: <1277426101.26161.168.camel@localhost>
References: <1276819235.14011.734.camel@localhost>
	<20100622223438.GA3898@kroah.com> <1277340122.26161.45.camel@localhost>
	<20100624222123.GC4838@kroah.com>
	<1277426101.26161.168.camel@localhost>
Date: Fri, 25 Jun 2010 01:35:49 +0100
Message-ID: <1277426149.26161.169.camel@localhost>
Mime-Version: 1.0
X-Mailer: Evolution 2.30.1.2 
X-SA-Exim-Connect-IP: 192.168.4.185
X-SA-Exim-Mail-From: ben@decadent.org.uk
X-Spam-Checker-Version: SpamAssassin 3.2.5 (2008-06-10) on
	shadbolt.decadent.org.uk
X-Spam-Status: No, score=-1.4 required=5.0 tests=ALL_TRUSTED
	autolearn=disabled version=3.2.5
Subject: [PATCH 2/2] Staging: rtl8192u_usb: Add LG device ID 043e:7a01
X-SA-Exim-Version: 4.2.1 (built Wed, 25 Jun 2008 17:14:11 +0000)
X-SA-Exim-Scanned: Yes (on shadbolt.decadent.org.uk)
Cc: devel@driverdev.osuosl.org,
 Florian Schilhabel <florian.c.schilhabel@googlemail.com>,
 Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>,
 Guy Sheffer <guysoft@gmail.com>
X-BeenThere: devel@linuxdriverproject.org
X-Mailman-Version: 2.1.12
Precedence: list
List-Id: Linux Driver Project Developer List <devel.linuxdriverproject.org>
List-Unsubscribe: <http://driverdev.linuxdriverproject.org/mailman/options/devel>,
	<mailto:devel-request@linuxdriverproject.org?subject=unsubscribe>
List-Archive: <http://driverdev.linuxdriverproject.org/pipermail/devel>
List-Post: <mailto:devel@linuxdriverproject.org>
List-Help: <mailto:devel-request@linuxdriverproject.org?subject=help>
List-Subscribe: <http://driverdev.linuxdriverproject.org/mailman/listinfo/devel>,
	<mailto:devel-request@linuxdriverproject.org?subject=subscribe>
Content-Type: text/plain;
  charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: devel-bounces@linuxdriverproject.org
Errors-To: devel-bounces@linuxdriverproject.org
X-GMX-Antivirus: 0 (no virus found)
X-GMX-Antispam: 0 (Mail was not recognized as spam);
 Detail=5D7Q89H36p6i75npGen84eVAEFK/syJmiNoEBJhgjYKpglu1TZLLw7xMZnJMXwBFK0zrU
 udEInhYyaWAzwtcf5K2pCdD+gZ2/z4PnBLkwixZI+pVtXqOlCN41sOWgjaVeH7+UhPxHlGxFK/rc
 sw7fg==V1;
X-GMX-UID: G8gRf2kXPTRtERAKKTIwJ+oxc2tpZAse
Status: R
X-Status: N
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  

Add another device ID as listed in the vendor driver version
0003.0825.2009.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 drivers/staging/rtl8192u/r8192U_core.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

--- a/drivers/staging/rtl8192u/r8192U_core.c
+++ b/drivers/staging/rtl8192u/r8192U_core.c
@@ -121,6 +121,8 @@ static const struct usb_device_id rtl819
 	{USB_DEVICE(0x2001, 0x3301)},
 	/* Zinwell */
 	{USB_DEVICE(0x5a57, 0x0290)},
+	/* LG */
+	{USB_DEVICE(0x043e, 0x7a01)},
 	{}
 };
 
